agenta-backend/agenta_backend/services/app_manager.py

```diff
@@ -103,7 +103,7 @@ async def start_variant(
         if isCloudEE():
             api_key = await api_key_service.create_api_key(
```

Comment:
What was the output of this function in the buggy version (seeing that we are calling it with a wrong workspace_id). Is there a way to add an assertion to make the `create_api_key` function throws an error with wrong inputs.

Reply:
Instead of the workspace ID being assigned to the api key, the workspace object/entity was being assigned.

Right behaviour:

```python
worspace_id = "q22e3rr"
```

Previous behaviour:

```python
workspace_id = "WorkspaceDB(id=ObjectId(...), owner=<Link at u3ej>, ...)"
```